### ISSUE TITLE
Error "Context Canceled" when pruning

### ISSUE THREAD

**Issue by fleshgolem** [ad-on-is/resticity#7]
I have set pruning rules up for all my repos and a schedule to run it manually. When i run it, I get this in the docker log

```
resticity  | 2024/03/19 15:43:15 DEBU Running job manually id=0c11b071-92e1-4daf-afd2-bc5e72ed9a5e
resticity  | 2024/03/19 15:43:15 DEBU before job run id=0c11b071-92e1-4daf-afd2-bc5e72ed9a5e
resticity  | 2024/03/19 15:43:15 DEBU Setting forced running job id=0c11b071-92e1-4daf-afd2-bc5e72ed9a5e
resticity  | 2024/03/19 15:43:15 INFO core repo=s3:xxx/xxx cmd=[unlock]
resticity  | 2024/03/19 15:43:17 INFO core repo=s3:xxx/xxx cmd="[forget --prune --keep-daily 7 --keep-weekly 8 --keep-monthly 10]"
resticity  | 2024/03/19 15:43:17 ERRO executing restic command err="context canceled"
resticity  | 2024/03/19 15:43:17 DEBU after job run res=success id=0c11b071-92e1-4daf-afd2-bc5e72ed9a5e
```
I know that this prune should delete some snapshots and I can verify that it does not. The UI will also show it as having run succesfully, even though it didnt
I have no idea, what the error "context canceled" means here, so I am not sure what other info to provide. Please let me know if you need anything else.

**Comment by ad-on-is** [comment]
Yep, can reproduce. Has something to do with unlocking the repository first. Bugfix is on its way

**Comment by ad-on-is** [comment]
Fixed in 1.0.32
